content/docs/setup/kubernetes/platform-setup/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Platform Setup
 description: How to prepare various Kubernetes platforms before installing Istio.
-weight: 1
+weight: 100
```

Comment:
On the contrary, Platform Setup should be last, I changed this on purpose.

First thing people need to see is the Quick Start page. They probably think their platform is ready. All quickstart and installation pages link to the platform setup and download of the release.

Besides, it is likely the actual quickstart or installation pages will be visited more often. Platform setup is done at most once.

Reply:
There are other prerequisites to installation, for instance Downloading the release , so it will be incorrect to call the platform setup section "preparing for installation".

In any case, further re-orgs can be done outside of this PR.

Reply:
Prerequisites should come first, before quick start, since due to prerequisites it ends up being neither a start or quick...

Reply:
Nack, platform setup is not the focus of the setup section. 

Downloading the release make sense to be the first.

Reply:
How about we go with my proposal and we get some feedback?

Reply:
Platform setup is an ancillary page shared by Quick Start and Helm. Is not required for Ansible, Helm or the other types of installation. It was extracted so we can reduce the length of the Quickstart page, and it does not deserve more attention. Ideally it should not even be in the menu on the left, but the website is limited in what can be achieved.

Download Release should be first, and then the actual installation pages.